packages/cubejs-client-core/src/tests/CubeApi.test.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * @license Apache-2.0
```

Comment:
```suggestion
 * @license MIT License
```

Reply:
Actually, feel free to remove this whole comment block

Reply:
Ok, got it.